electron/js/lib/googleAuth.js

```diff
@@ -41,12 +40,13 @@ const _authorizeApp = (url) => {
       setImmediate(() => {
         const title = win.getTitle();
 
+        const [, , returnValue] = title.split(/[ =]/);
```

Comment:
is this really better than ```const returnValue = title.split(/[ =]/)[2];```?

Reply:
But it fails as soon as you enable magic number checking